.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+	"bracketSpacing": true,
+	"printWidth": 80,
```

Comment:
Don't we usually use 120?

Reply:
[Yeah](https://github.com/search?q=org%3Aopenmrs%20printWidth&type=code). I'll switch back to that.